pkg/controller/multiarchbuildconfig/multiarchbuildconfig.go

```diff
@@ -121,37 +120,44 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, logge
 }
 
 func (r *reconciler) handleMultiArchBuildConfig(ctx context.Context, mabc *v1.MultiArchBuildConfig) error {
+	var mutateFn func(mabcToMutate *v1.MultiArchBuildConfig)
+
 	if mabc.Status.Builds == nil {
 		if err := r.createBuildsForArchitectures(ctx, mabc); err != nil {
 			return fmt.Errorf("couldn't create builds for architectures: %s: %w", strings.Join(r.architectures, ","), err)
 		}
+		if err := v1.UpdateMultiArchBuildConfig(ctx, r.logger, r.client, ctrlruntimeclient.ObjectKey{Namespace: mabc.Namespace, Name: mabc.Name}, mutateFn); err != nil {
```

Comment:
This won't update anything. Also, keep in mind that there is another controller that watches the builds and will update the mabc.Status.Builds either way.

Reply:
Oh yeah, copy-paste mistake I guess.
> there is another controller that watches the builds

Yes I know but we can probably get rid of it and use the owner references instead. I'll open a follow up PR.